bench.jl

```diff
@@ -0,0 +1,16 @@
+using Arrow, ArrowTypes, Random, BenchmarkTools
```

Comment:
do we want to check this in at top-level? could have a `benchmark` folder

Reply:
if we're keeping this, we need to add the license block to the top to pass the CI audit check